x/stake/mock/validator.go

```diff
@@ -0,0 +1,122 @@
+package mock
```

Comment:
Can we put this in `democoin` instead of in `x/stake`?

Reply:
Also I'm not sure we really need to implement `sdk.ValidatorSet` for democoin.

Reply:
Just for testing purpose - much simpler to add/remove validator from the validatorset.

Reply:
OK, but let's still move to `democoin`, I think we want to keep any nonessential code out of the core modules.